src/basic/fd-util.h

```diff
@@ -101,6 +102,9 @@ int read_nr_open(void);
 int fd_get_diskseq(int fd, uint64_t *ret);
 
 int dir_fd_is_root(int dir_fd);
+static inline int dir_fd_is_root_or_cwd(int dir_fd) {
```

Comment:
Commit message says "install dir_fd_is_root_or_cwd()", should be "introduce dir_fd_is_root_or_cwd()" I guess?